index.html

```diff
@@ -347,9 +151,70 @@ <h3>Skills</h3>
       <img id="indicator" src="assests/images/shape.png" alt="Home Indicator/Light">
     </div>
   </footer>
-
+  
+    <!-- Popup Window -->
+    <div class="modal-overlay">
```

Comment:
- [ ] On mobile view I have realized that  the design does not match the chosen Figma design look. This is mainly caused by making use of the wrong margins/ padding to space and style this section from the margins of the popup. Kindly look into this.

**This is how I am seeing the section:**

[Screencast from 23-11-2022 08:54:00 ALASIRI.webm](https://user-images.githubusercontent.com/86318284/203615974-78a55305-6f90-4528-8312-81d635ed109d.webm)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/203616090-910ebe7a-033e-4d65-a79e-e84c678e1328.png)